README.md

```diff
@@ -87,6 +87,37 @@ print(list(sorted(ng.optimizers.registry.keys())))
 
 The [optimization documentation](docs/optimization.md) contains more information on how to use several workers, take full control of the optimization through the `ask` and `tell` interface and some pieces of advice on how to choose the proper optimizer for your problem.
 
+
+## Example of chaining, or inoculation, or initialization of an evolutionary algorithm
```

Comment:
Please move this to docs/optimization.md
The main README is only for a brief look at the AP and to give pointers for more